marklogic-client-api/src/main/java/com/marklogic/client/expression/PlanBuilderBase.java

```diff
@@ -464,6 +464,18 @@ public interface PlanBuilderBase {
      */
     TransformDefinition transformDefinition(String path);
 
+    /**
+     * Build a schema definition for use with {@code validateDoc}.
+     *
+     * @return a new {@code SchemaDefExpr}
+     */
+    SchemaDefExpr schemaDefinition(String kind);
+    /**
+     * Build a error definition for use with {@code validateDoc}.
+     *
+     * @return a new {@code PlanErrorDisposition}
```

Comment:
Since logLevel can be null, the javadocs should have "param"'s defined that explains that logSize is required and logLevel is null. Also need to define what are valid values for each of them.

Reply:
i tried without loglevel and it is throwing error. Is it a bug then?

Reply:
The client should still provide a logLevel if the user doesn't. However, see our chat room today - I don't think we're going to support this because it's unclear what "summary" and "detail" should even mean.

Reply:
okay. then i will revisit this as well :-)